src/bluesearch/widgets/search_widget.py

```diff
@@ -936,9 +935,8 @@ def _cb_bt_pdf_report_search(self, change_dict):
 
             css_style = style.get_css_style()
 
-            pdfkit.from_string(
-                f"<style> {css_style} </style>"
-                + hyperparameters_section
-                + results_section,
-                f"report_{datetime.datetime.now()}.pdf",
-            )
+            output_file = pathlib.Path(f"report_{datetime.datetime.now()}.html")
+            with output_file.open("w") as f:
+                f.write(f"<style> {css_style} </style>")
+                f.write(hyperparameters_section)
+                f.write(results_section)
```

Comment:
So this works without including the `<!DOCTYPE html>`, `<html>` etc. tags?

Reply:
1. `<!DOCTYPE html>` is a special beast,  and seems to be required for for legacy reasons. Modern browsers should not have issues even if it's omitted, but I think there's nothing harmful in adding it. Done in 158935e.
2. `<html>`, `<head>` ... can apparently be omitted as per [HTML specs](https://html.spec.whatwg.org/multipage/syntax.html#syntax-tag-omission).